packages/odyssey-react/src/components/TextArea/TextArea.test.tsx

```diff
@@ -109,5 +109,11 @@ describe("TextArea", () => {
     expect(ref).toHaveBeenLastCalledWith(screen.getByRole(textBox));
   });
 
+  it("assigns resize prop correctly", () => {
+    render(<TextArea resize="none" label={label}/>);
+    const container = screen.getByRole(textBox, { name: label });
+    expect(container).toHaveClass('noneResize');
```

Comment:
a slightly better test would involve the `toHaveStyle` assertion: https://github.com/testing-library/jest-dom#tohavestyle

Reply:
CSS never complied in test suite and current test should be ok.